apps/server/src/modules/files-storage/service/files-storage-delete.service.spec.ts

```diff
@@ -0,0 +1,274 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { MikroORM } from '@mikro-orm/core';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { InternalServerErrorException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { AntivirusService } from '@shared/infra/antivirus/antivirus.service';
+import { fileRecordFactory, setupEntities } from '@shared/testing';
+import { Logger } from '@src/core/logger';
+import { S3ClientAdapter } from '../client/s3-client.adapter';
+import { FileRecordParams } from '../controller/dto';
+import { FileRecordParentType } from '../entity';
+import { getPaths } from '../helper';
+import { FileRecordRepo } from '../repo';
+import { FilesStorageService } from './files-storage.service';
+
+const getFileRecordsWithParams = () => {
```

Comment:
I think this should be called "buildFileRecordsWithParams". "get" sounds like getting it from the database or somewhere.

And parentId is returned twice, in params and as separate prop. On purpose?